app/src/main/java/org/schabi/newpipe/CheckForNewAppVersion.java

```diff
@@ -218,15 +208,14 @@ public static Disposable checkNewVersion(@NonNull final App app) {
                             } catch (final JsonParserException e) {
                                 // connectivity problems, do not alarm user and fail silently
                                 if (DEBUG) {
-                                    Log.w(TAG, Log.getStackTraceString(e));
+                                    Log.w(TAG, "Could not get NewPipe API: invalid json", e);
                                 }
                             }
                         },
-                        throwable -> {
+                        e -> {
                             // connectivity problems, do not alarm user and fail silently
                             if (DEBUG) {
-                                Log.i(TAG, "Could not get NewPipe API: network problem");
-                                Log.i(TAG, Log.getStackTraceString(throwable));
+                                Log.w(TAG, "Could not get NewPipe API: network problem", e);
```

Comment:
i just thought about checking whether there was really a network problem by calling `ExceptionUtils.isNetworkRelated(e)`

Reply:
What else can it be? And even if the error was different, I don't think it would be an issue, since the whole stacktrace would be printed out anyway

Reply:
ok